src/Symfony/Component/Messenger/Bridge/AmazonSqs/Middleware/AddFifoStamp.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\AmazonSqs\Middleware;
+
+use Symfony\Component\Messenger\Bridge\AmazonSqs\Transport\AmazonSqsFifoStamp;
+use Symfony\Component\Messenger\Envelope;
+use Symfony\Component\Messenger\Middleware\MiddlewareInterface;
+use Symfony\Component\Messenger\Middleware\StackInterface;
+
+final class AddFifoStamp implements MiddlewareInterface
+{
+    public function handle(Envelope $envelope, StackInterface $stack): Envelope
+    {
+        $message = $envelope->getMessage();
+        $messageGroupId = null;
+        $messageDeduplicationId = null;
+
+        if ($message instanceof SupportsMessageGroupInterface) {
+            $messageGroupId = $message->messageGroupId();
```

Comment:
messageGroupId or getMessageGroupId? any reason why you choose this style that relates to the codebase?

Reply:
let's rename then, for consistency with the existing codebase in this domain